test/bin/test

```diff
@@ -0,0 +1,80 @@
+#! /bin/sh -eu
+#
+# [<php-version>] [<dependency-preference>]
+#
+# <php-version>  example: php70
+# <dependency-preference> One of highest (default), lowest
+#
+
+dcexec="docker-compose exec -u `id --user`"
```

Comment:
This fails on MacOS, but just ```id -u``` does the same and works on both.

Reply:
Thanks you for this feedback.